frontend/src/components/pages/components/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,143 @@
+import { Accordion, Nav, Navbar, Container } from 'react-bootstrap';
+import PlusButton from './plusButton';
+import styles from './styles.module.scss';
+
+const Toolbar: React.FC = () => {
+
+  return (
+    <>
+      <Container className={styles.toolbarContainer}>
+
+        <h2 className={styles.logoToolbar}>Infostack</h2>
+        <p className={styles.sectionName}>Pages</p>
+        <Accordion className={styles.accordion} defaultActiveKey="1" flush>
+
+          <Accordion flush>
+            <Accordion.Item eventKey="0" className={styles.accordionItem}>
+              <Nav.Link className={styles.navbarBrand}>Dashboards</Nav.Link>
+            </Accordion.Item>
+          </Accordion>
+
+          <Accordion.Item eventKey="1" className={styles.accordionItem}>
+            <Accordion.Header className={styles.accordionHeader}><Navbar.Brand className={styles.greyTextColor}>Pages</Navbar.Brand></Accordion.Header>
+            <Accordion.Body className={styles.accordionBody}>
+
+              <Accordion flush>
+                <Accordion.Item eventKey="0" className={`${styles.accordionItem} ${styles.accordionItemInsideSection}`}>
```

Comment:
Please use `getAllowedClasses` helper when you need to concatenate several classes.
```suggestion
                <Accordion.Item eventKey="0" className={getAllowedClasses(styles.accordionItem, styles.accordionItemInsideSection)}>
```